dockerplugin.py

```diff
@@ -90,22 +90,34 @@ def read_blkio_stats(container, dimensions, stats, t):
         # numbers). We need to group and report the stats of each block
         # device independently.
         blkio_stats = {}
+        blkio_major_stats = {}
+        blkio_minor_stats = {}
+
         for value in values:
+
             k = '{key}-{major}-{minor}'.format(key=key,
```

Comment:
as long as there isn't case of same key with different block device major/minor values.
In that case the current way works correctly.  But if that is never that case then I will remove the major and minor from key.  I'll circle back and see